Data Plane logs (#934)
Kong/developer.konghq.com: app/konnect-logs.md

    url: /gateway/logs/
---

## Data Plane logs

> Data plane logs provide users with the ability to view, filter, search, and download logs for the Data Plane nodes associated with a Dedicated Cloud Gateway. This feature is essential for debugging, monitoring, and understanding the behavior of Data Plane nodes.

Reviewer: ```suggestion
Data Plane logs provide users with the ability to view, filter, search, and download logs for the Data Plane nodes associated with a Dedicated Cloud Gateway. 
Reviewing logs is essential for debugging, monitoring, and understanding the behavior of Data Plane nodes.
```

> ## Log Management Features

Reviewer: ```suggestion
## Log management in Gateway Manager
```
Can we please avoid calling things "features"? We fall into the trap of documenting each feature rather than capability/purpose that way

> You can access Data Plane logs from the Data Plane Dashboard in [Gateway Manager](https://cloud.konghq.com/us/gateway-manager/).

Reviewer: ```suggestion
You can access Data Plane logs from the Data Plane dashboard in [Gateway Manager](https://cloud.konghq.com/us/gateway-manager/).
```

> From here, you can perform the following actions:

Reviewer: ```suggestion
From the Data Plane dashboard, you can perform the following actions:
```
More useful for search.

{% table %}
columns:

>   - title: Feature

Reviewer: ```suggestion
  - title: Use Case
```

    key: feature
  - title: Description
    key: description
rows:
  - feature: View Logs
    description: Access detailed logs for all Data Plane nodes in your gateway.
  - feature: Filter Logs

>     description: Type keywords or phrases in the Filter Log Messages box to refine log content.

Reviewer: ```suggestion
    description: Type keywords or phrases in the **Filter Log Messages** box to refine log content.
```

  - feature: Download Logs

>     description: Click the Download Icon near the log table to save logs locally.

Reviewer: ```suggestion
    description: Click the download icon near the log table to save logs locally.
```

  - feature: Data Plane Node Selection

>     description: Use the Node ID input box to view logs from a specific Data Plane node.

Reviewer: ```suggestion
    description: Use the **Node ID** input box to view logs from a specific Data Plane node.
```

  - feature: Date Range Selection

>     description: Use the **Date Picker tool to specify a date range for logs.

Reviewer: ```suggestion
    description: Use the date picker tool to specify a date range for logs.
```
